t/multimail.expect

```diff
@@ -3358,3 +3358,111 @@ To stop receiving notification emails like this one, please contact
 Administrator <administrator@example.com>.
 EOF
 ######################################################################
+*** no recipients configured so no email will be sent
+*** for 'refs/heads/formatting' update 6522cd3759908339b14ab0048bc1d756929ac112->8e11f7c6a6a86d37ac1f3e9ce6ec7d1399412c17
+Add Some One <Some.One@example.com>, Another Guy <Another.Guy@example.com>, Third.Guy@example.com to CC for 919426837aa07c18459a0acc0f789b6a8cd8fb80
+######################################################################
+/usr/sbin/sendmail -oi -t -f Sender <sender@example.com> <<EOF
+Date: ...
+To: 
+Cc: Some One <Some.One@example.com>, Another Guy <Another.Guy@example.com>,
```

Comment:
It feels weird to have an empty To: field an only recipients in Cc:. Do you think it would make sense to put everyone in To: and have no Cc: in this case?